lib/tasks/avalon_ualbertalib.rake

```diff
@@ -66,6 +68,177 @@ namespace :avalon do
+          puts "Manditory argument: -c id"
+          puts option_parser.help
+          exit 1
+        end
+        
+        obj = ActiveFedora::Base.find(options[:id])
+        if (obj.is_a?(Admin::Collection))
+          obj.media_objects.to_a.collect.each do |m| 
+            puts "%s" % [m.id] 
+          end;
+        else
+          puts "ERROR: not a Collection obj: [%s]" % [options[:id]] 
+        end
+
+      end
+ 
+      desc "Given a set of IDs, in a file, one-per-line, export contents"
+      task :export_contents => :environment do
+
+        options = {}
+        option_parser = OptionParser.new do |opts|
+          opts.on("-m=ARG", "--objects=ARG", "File containing list of object IDs, one per line") { 
+            |filename| options[:filename] = filename 
+          }
+          opts.on("-d=ARG", "--derivatives=ARG", "Base directory of the derivatives") { 
+            |stream_base_dir| options[:stream_base_dir] = stream_base_dir 
+          }
+          opts.on("-b=ARG", "--dest_base=ARG", "Output directory") { 
+            |dest_base_dir| options[:dest_base_dir] = dest_base_dir 
+          }
+          opts.on("-s=ARG", "--delay=ARG", "Delay between exports") { 
+            |delay| options[:delay] = delay ? delay.to_i : 0
+          }
+          opts.on("-h", "--help") do 
+             puts opts
+             exit 
+          end
+        end
+        
+        args = option_parser.order!(ARGV) {}
+        option_parser.parse!(args)
+
+        if options[:filename].nil?
+          puts "Manditory argument: -m filename"
+          puts option_parser.help
+          exit 1
+        elsif !File.exist?(options[:filename])
+          puts "Filename not found: #{options[:filename]}"
+          exit 1
+        end
+
+        ids = File.read(options[:filename]).split
+
+        count = 0
+        count_success = 0
+        ids.each do |id|
+          begin  
+            count += 1
+            obj = ActiveFedora::Base.find(id)
+            if (obj.is_a?(MediaObject))
+              ualberta_export_mediaObject(obj, options)
+            elsif (obj.is_a?(Admin::Collection))
+              ualberta_export_collectionObject(obj, options)
+            end
+            count_success += 1
+            sleep(options[:delay]) if (options[:delay])
+          rescue StandardError => e
+            puts "ERROR: %s - %s - %s" % [id, e.class.name, e.message]
+            puts e.backtrace
+          end
+        end
+        puts "Number processed: [#{count}]; success: [#{count_success}]"
+        if count == 0
+          puts "You must specify an set of MediaObject IDs."
+          puts option_parser.help
+          exit 1
+        end
+      end
+
+      def ualberta_export_collectionObject( obj, options )
+        base_dir = File.join(options[:dest_base_dir], obj.id)
+        FileUtils.mkdir_p(base_dir)
+        File.open("#{base_dir}/collection_object.json", 'w') { |file| file.write(obj.as_json.to_json(:except => :captions)) }
```

Comment:
The hash rocket (`key => value`) syntax was common before Ruby 1.9.  I think the preferred form is now `key: value`
```suggestion
        File.open("#{base_dir}/collection_object.json", 'w') { |file| file.write(obj.as_json.to_json(except: :captions)) }
```